### PR TITLE
Document Battle Frontier Outside, Ranking Hall, and Scotts House

### PR DESCRIPTION
**Pull request by GriffinRichards**

### CODE REVIEW

File: src/battle_tower.c [hunk 1]
```diff
@@ -1051,24 +1052,24 @@ struct
...
+    [BATTLE_TOWER_FUNC_0] = sub_8161F94,
+    [BATTLE_TOWER_FUNC_1] = sub_8162054,
+    [BATTLE_TOWER_FUNC_2] = sub_81620F4,
+    [BATTLE_TOWER_FUNC_CHOOSE_TRAINER] = ChooseNextBattleTowerTrainer,
+    [BATTLE_TOWER_FUNC_4] = sub_81621C0,
+    [BATTLE_TOWER_FUNC_GIVE_RIBBONS] = AwardBattleTowerRibbons,
+    [BATTLE_TOWER_FUNC_SAVE] = SaveBattleTowerProgress,
+    [BATTLE_TOWER_FUNC_7] = sub_8163914,
+    [BATTLE_TOWER_FUNC_NOP] = nullsub_61,
+    [BATTLE_TOWER_FUNC_NOP_SPRITE] = SpriteCB_Null6,
```
Comment (huderlem) [inline src/battle_tower.c]: This function is not related to a sprite or sprite callback.

Comment (GriffinRichards) [reply src/battle_tower.c]: Seems these were introduced by #553, probably worth removing the others that also aren't sprite callbacks. In any case this ones fixed
